smart:
  hardware_interface:
     loop_hz: 50 # hz
     joints:
       - front_left_wheel_joint
       - front_right_wheel_joint
       - middle_left_wheel_joint
       - middle_right_wheel_joint
       - rear_left_wheel_joint
       - rear_right_wheel_joint
       - front_left_wheel_steering_joint
       - front_right_wheel_steering_joint
       - middle_left_wheel_steering_joint
       - middle_right_wheel_steering_joint
       - rear_left_wheel_steering_joint
       - rear_right_wheel_steering_joint
       - camera/z
       - camera/y
